spec/spec-helper.coffee

```diff
@@ -1,5 +1,12 @@
 require 'coffee-cache'
+Point = require '../src/point'
 
 exports.expectValues = (actual, expected) ->
   for expectedKey, expectedValue of expected
     expect(actual[expectedKey]).toEqual expectedValue
+
+exports.expectMappings = (mappings, {fromLayer, toLayer}) ->
```

Comment:
I'm confused about how you're able to access this without explicitly importing it in each spec. What am I missing?